tsl/src/compression/compress_utils.c

```diff
@@ -320,8 +320,12 @@ tsl_compress_chunk_wrapper(Oid chunk_relid, bool if_not_compressed)
 Datum
 tsl_compress_chunk(PG_FUNCTION_ARGS)
 {
-	Oid uncompressed_chunk_id = PG_ARGISNULL(0) ? InvalidOid : PG_GETARG_OID(0);
+	Oid uncompressed_chunk_id = PG_GETARG_OID(0);
```

Comment:
I don't think the previous code was worse and I see no value in removing the NULL-check. In contrast, the code is no longer protected against changes in the `STRICT` property in the function declaration. Also, how is this related to other changes?

If you want to assert, you can still `Assert(OidIsValid(uncompressed_chunk_id))` below.

Reply:
The function is declared strict so I see little value in having additional checks and treating `InvalidOid` as an acceptable value for further processing, it just suggest that the function can handle NULL values. It is also not was PostgreSQL itself is doing. However, I removed this from the commit since is not important for the main issue of this commit.

Reply:
The function is declared strict so I see little value in having additional checks and treating `InvalidOid` as an acceptable value for further processing, it just suggest that the function can handle NULL values. It is also not was PostgreSQL itself is doing. However, I removed this from the commit since is not important for the main issue of this commit.